user-guide/Advanced_Modules/Dashboards_app/Dashboard_Gateway_installation.md

```diff
@@ -67,3 +71,100 @@ There are two main reasons to consider a Dashboard Gateway setup:
+
+1. Open IIS Manager on the Gateway server.
+
+1. In the tree view on the left, choose a server node and select the *Application Request Routing* feature.
+
+1. Select the *Enable Proxy* checkbox.
+
+1. All the other settings can be left on default.
+
+#### Creating rewrite rules for the Maps module
+
+To add a rewrite rule that will proxy any request to the DataMiner Maps module at `https://mydma/maps/`, as long as the requested URL path starts with "maps", do the following:
+
+1. Open IIS Manager on the Gateway server.
+
+1. In the tree view on the left, choose a web site node and select the *URL Rewrite* feature.
+
+1. Click *Add Rule(s)*, select *Blank rule*, and specify the following options:
+
+   - **Name**: e.g. "Maps module"
+
+   - **Pattern**: `^maps/(.*)`
+
+   - **Rewrite URL**: `https://mydma/maps/{R:1}`
+
+   - ***Stop processing of subsequent rules***: enabled
+
+1. Click *apply*.
+
+To add a rewrite rule that will proxy any request to the Maps API at `https://mydma/API/v0/maps.asmx`, as long as the requested URL path starts with "API/v0/maps.asmx", do the following:
+
+1. Open IIS Manager on the Gateway server.
+
+1. In the tree view on the left, choose a web site node and select the *URL Rewrite* feature.
+
+1. Click *Add Rule(s)*, select *Blank rule*, and specify the following options:
+
+   - **Name**: e.g. "Maps API"
+
+   - **Pattern**: `^API/v0/maps.asmx/(.*)`
+
+   - **Rewrite URL**: `https://mydma/API/v0/maps.asmx/{R:1}`
+
+   - ***Stop processing of subsequent rules***: enabled
+
+1. Click *apply*.
+
+#### Creating rewrite rules for the VideoThumbnails web page
+
+To add a rewrite rule that will proxy any request to the VideoThumbnails web page at `https://mydma/videothumbnails/`, as long as the requested URL path starts with "videothumbnails", do the following:
+
+1. Open IIS Manager on the Gateway server.
+
+1. In the tree view on the left, choose a web site node and select the *URL Rewrite* feature.
+
+1. Click *Add Rule(s)*, select *Blank rule*, and specify the following options:
+
+   - **Name**: e.g. "VideoThumbnails"
+
+   - **Pattern**: `^videothumbnails/(.*)`
+
+   - **Rewrite URL**: `https://mydma/videothumbnails/{R:1}`
+
+   - ***Stop processing of subsequent rules***: enabled
+
+1. Click *apply*.
+
+### Testing the reverse proxy functionality
+
+To test whether the reverse proxy is working properly, do the following:
+
+1. Enter `https://gateway/maps/maps.aspx?config=yourmapconfig` or `https://gateway/videothumbnails/video.htm` in your browser's address bar.
+
+1. If the reverse proxy was configured correctly, you should see the login screen of the DataMiner Maps module or the VideoThumbnails web page. It should be possible to log in.
+
+> [!TIP]
+> When embedding the URLs in dashboards, make use of relative URLs, e.g. ``/maps/maps.aspx?config=yourmapconfig``. This way, the content will both be shown on a dashboard accessed via the Gateway server and a dashboard accessed locally via the DMA.
```

Comment:
During which step are the users embedding URLs in dashboards? Can this tip move to a more suited place in this section of the documentation, instead of under "Testing the reverse proxy functionality"?

Reply:
I will move the tip to the beginning of the section on Reverse proxy so that it is hopefully a bit more clear that this is a general tip that comes along with Maps/VideoThumbnails in Dashboards and that it is not specific to the final section on testing the reverse proxy functionality.